src/Microsoft.AspNet.Mvc.Core/Filters/ResponseCacheAttribute.cs

```diff
@@ -38,7 +39,17 @@ public int Duration
         /// <summary>
         /// Gets or sets the location where the data from a particular URL must be cached.
         /// </summary>
-        public ResponseCacheLocation Location { get; set; }
+        public ResponseCacheLocation Location
```

Comment:
I have this property backed by a nullable _location to know if this was set in the attribute. If it was set, I can use this in identifying if I need to override the cache profile or not.